kalite/distributed/static/js/distributed/video/views.js

```diff
@@ -60,7 +60,16 @@ var VideoPlayerView = ContentBaseView.extend({
         var player_id = this.$(".video-js").attr("id");
 
         if (player_id) {
-            this.player = this.player = _V_(player_id);
+            var video_player_options = {
+                "controls": true,
+                "width": "auto",
+                "height": "auto",
+                "playbackRates": [0.5, 1, 1.5, 2]
```

Comment:
Not sure what it supports, but I've found 1.25 can be a pleasant playback speed for a lot of content as well.